pkg/skaffold/runner/runner.go

```diff
@@ -35,7 +35,7 @@ type Runner interface {
 	Apply(context.Context, io.Writer) error
 	ApplyDefaultRepo(tag string) (string, error)
 	Build(context.Context, io.Writer, []*latest.Artifact) ([]graph.Artifact, error)
-	Cleanup(context.Context, io.Writer, bool, manifest.ManifestList) error
+	Cleanup(context.Context, io.Writer, bool, manifest.ManifestList, *manifest.ManifestListByConfig) error
```

Comment:
do you need both parameters `manifest.ManifestList` and `*manifest.ManifestListByConfig`?

Reply:
Can we try removing `manifest.ManifestList` as a parameter in this PR itself? Otherwise it just looks weird. I don't think that there should be any reference of `manifest.ManifestList` that can't use `manifest.ManifestListByConfig`

Reply:
done